executor/src/witgen/mod.rs

```diff
@@ -418,7 +546,7 @@ impl<'a, T: FieldElement> FixedData<'a, T> {
                 .definitions
                 .iter()
                 .filter(|(_, (symbol, _))| matches!(symbol.kind, SymbolKind::Poly(_)))
-                .map(|(name, (symbol, _))| (name.clone(), symbol.into()))
+                .flat_map(|(_, (symbol, _))| symbol.array_elements())
```

Comment:
This was a bug IMO, didn't expand array elements.

Reply:
Maybe Symbol should not implement this Into, or at least change to TryInto

Reply:
Agree, but this can be a different PR. I just checked and it is only used several times in `Analyzed::remove_definitions`.